cmd/protoc-gen-fastmarshal/templates/fieldsnippets.tmpl

```diff
@@ -490,12 +499,14 @@
 {{- $mapKeyKind := (.Desc.MapKey.Kind | string) -}}
 {{- $mapValueKind := (.Desc.MapValue.Kind | string) -}}
     for k, v := range m.{{.GoName | getSafeFieldName}} {
-        var l int
         {{ if eq $mapValueKind "int32" "int64" "uint32" "uint64" "enum" -}}
         itemSize := 1 + csproto.SizeOfVarint(uint64(v))
+        {{- else if eq $mapValueKind "bool" -}}
+        _ = v
+        itemSize := 1 + 1
         {{- else if eq $mapValueKind "string" "bytes" -}}
-        l = len(v)
-        itemSize := 1 + csproto.SizeOfVarint(uint64(l)) + l
+        valueSize := len(v)
```

Comment:
Nit, but why declare a new variable here?  Same for the other spots below.

Reply:
Because in many of the cases `l` was not used at all, therefore I removed it from every iteration and only create it when needed.